packages/api/src/wallets/NFT.ts

```diff
@@ -33,13 +33,25 @@ export default class NFTWallet extends Wallet {
     });
   }
 
-  async setNftDid(walletId: number, nftCoinId: string, did: string, fee: string) {
-    return this.command('nft_set_nft_did', {
-      walletId,
-      nftCoinId,
-      didId: did,
-      fee,
-    });
+  async setNftDid(walletId: number, nftCoinIds: string[], did: string, fee: string) {
+    if (nftCoinIds.length === 1) {
+      return this.command('nft_set_nft_did', {
+        walletId,
+        nftCoinId: nftCoinIds[0],
+        didId: did,
+        fee,
+      });
+    }
+    return Promise.all(
+      nftCoinIds.map((nftCoinId) =>
+        this.command('nft_set_nft_did', {
+          walletId,
+          nftCoinId,
+          didId: did,
+          fee,
+        })
+      )
+    ).then((responses) => responses);
```

Comment:
I may have miscommunicated the RPC changes we need to support bulk NFT DID assignment. We should be calling `nft_set_nft_did` for assigning a DID to a single NFT, but for assigning a DID to _multiple_ NFTs, we should instead be calling the new RPC `nft_set_did_bulk`

So we should have a new function like:
```
async setNftDidBulk(nftCoinList: string[], fee: string, didId: string | undefined) {
    ...
    this.command('nft_set_did_bulk',  {
        nftCoinList,
        didId,
        fee,
    })
    ...
```

The RPC parameters are named `nftCoinList`, `didId`, and `fee`

It's valid for `didId` to be undefined, in which case, the DID will be removed from the specified NFTs.